src/Microsoft.AspNetCore.Components.WebAssembly.DebugProxy/Microsoft.AspNetCore.Components.WebAssembly.DebugProxy.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>

```

Comment:
```suggestion
    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Reply:
You should copy this https://github.com/dotnet/aspnetcore/blob/master/src/Components/WebAssembly/Compression/src/runtimeconfig.template.json over to this repo. This would allow the app to use a 5.0 runtime if it's the only one installed